index.bs

```diff
@@ -504,7 +500,7 @@ It must act as follows:
     <li>Else, let |relativeStart| be <code>min(start, size)</code>.
   </ol>
 
-3. The optional <dfn argument for="Blob/slice(start, end, contentType)" id="dfn-end">end</dfn> parameter
+2. The optional <dfn argument for="Blob/slice(start, end, contentType)" id="dfn-end">end</dfn> parameter
```

Comment:
Since the spec is already inconsistent about use of 1/1/1/... vs. 1/2/3/..., how about converting this to 1/1/1/... while you're here to save work later?